app/views/users/show.html.erb

```diff
@@ -1,4 +1,32 @@
-<div>
-  <h1><%= @user.name %></h1>
-  <p><%= @user.bio %></p>
+<div class='user_div'>
+  <div class='user_photo'>
+    <%= image_tag @user.photo %>
+  </div>
+  <div class='user_info'>
+    <p class='user_name'><%= @user.name %></p>
+    <p class='number_of_posts'>Number of posts: <%= @user.post_counter %></p>
+  </div>
+</div>
+
+<div class='user_bio'>
+  <%= @user.bio %>
+</div>
+
+<div class='posts'>
+  <% post_number = 1 %>
+  <% @posts.each do |post| %>
+    <div class='individual_post'>
+      <h2>Post #<%= post_number %></h2>
+      <h3><%= post.title %></h3>
```

Comment:
- [ ] Kindly make sure to use the created methods in the model (`most_recent_post`), as per [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Use%20methods%20that%20you%20have%20created%20in%20the%20Blog%20App%20%2D%20processing%20data%20in%20models.) requirement.